environments/environments.py

```diff
@@ -17,10 +17,10 @@
         "USE_SPACY_TOKENIZER": 1,
         "FREEZE_EMBEDDINGS": ["VAMPIRE"],
         "EMBEDDINGS": ["VAMPIRE", "RANDOM"],
-        "ENCODER": "AVERAGE",
-        "EMBEDDING_DROPOUT": 0.5,
-        "LEARNING_RATE": 0.004,
-        "DROPOUT": 0.5,
+        "ENCODER": "CNN",
```

Comment:
Classifier environment by default will now be CNN; was this intentional?

Reply:
This was a typo! Good catch